File: AppliIHM/ihmSupervision/boitier.cpp
#include "boitier.h"

Boitier::Boitier()
{

}

Boitier::Boitier(std::string nomBoitier)
{
    this->nom = nomBoitier;
}


std::string Boitier::getNom()
{
    return this->nom;
}

void Boitier::setNom(std::string nom)
{

    this->nom = nom;
}

File: Web/siteClient/modele/Boitier.php
<?php

class Boitier {
	
	public $id;
	public $nom;
	public $hauteur;
	public $matiere;

	
	public function __construct($id)
	{
		$this->id = $id;
		$this->nom = "";
		$this->matiere = "";
		$this->hauteur = "";
	}

}